docs/warnings.rst

```diff
@@ -27,7 +27,7 @@ script using the `warnings.filterwarnings` function as follows::
      >>> import warnings
      >>> from astropy.io import fits
      >>> warnings.filterwarnings('ignore', category=UserWarning, append=True)
-     >>> fits.writeto(filename, data, clobber=True)
```

Comment:
This could also cover other kinds of warnings too though, for example FITS validation warnings for the header values? We definitely can't use clobber anymore if it's being removed.